Error: Webview API guide: TypeScript error in the "Controlling access to local resources" example

The example in the [Controlling access to local resources](https://code.visualstudio.com/api/extension-guides/webview#controlling-access-to-local-resources) section contains the following error:

```
src/extension.ts:22:52 - error TS2345: Argument of type 'string' is not assignable to parameter of type 'Uri'.

localResourceRoots: [vscode.Uri.joinPath(context.extensionPath, 'media')]
                                         ~~~~~~~~~~~~~~~~~~~~~
Found 1 error. Watching for file changes.
```

The error occurs because `localResourceRoots` expects an array of vscode URIs, but the `context.extensionPath` passes the path to the extension's directory as a string.

--- a/api/extension-guides/webview.md
+++ b/api/extension-guides/webview.md
@@ -496,9 +496,7 @@ export function activate(context: vscode.ExtensionContext) {
         vscode.ViewColumn.One,
         {
           // Only allow the webview to access resources in our extension's media directory
-          localResourceRoots: [
-            vscode.Uri.joinPath(context.extensionPath, 'media')
-          ]
+          localResourceRoots: [vscode.Uri.joinPath(context.extensionUri, 'media')]
         }
       );
 

